src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -162,4 +181,17 @@ private void AddUnmatchedSourceMembersDiagnostics()
             );
         }
     }
+
+    private void AddUnmatchedParametersDiagnostics()
```

Comment:
With the current architecture, if the user defined mapping is not an object mapping (eg. string to int), RMG048 is not emitted.